action.yml

```
@@ -5,15 +5,19 @@ branding:
 description: Runs OctoGuide as a GitHub Action. 🗺️
 inputs:
   comment-footer:
-    description: Custom footer text for bot comments.
+    description: Custom footer text for OctoGuide bot comments.
     required: false
   comment-header:
-    description: Custom header text for bot comments.
+    description: Custom header text for OctoGuide bot comments.
     required: false
   config:
     default: recommended
     description: Which config to use, as 'recommended' (default), 'strict', or 'none'.
     required: false
+  exclude-bots:
+    default: "true"
```

[Refactor] I personally find it confusing when the default value for an option is truthy. It's not just that it's an extra line for setting the default. It's also that conceptually I think simplest API norms leans toward _"if something isn't provided, it doesn't exist"_. WDYT about inverting to `include-bots` so the default value is falsy?

```suggestion
  include-bots:
```